OpenRA.Game/Graphics/TerrainRenderer.cs

```diff
@@ -9,25 +9,34 @@
 #endregion
 
 using System;
+using System.Collections.Generic;
 using OpenRA.Traits;
 
 namespace OpenRA.Graphics
 {
 	sealed class TerrainRenderer : IDisposable
 	{
-		readonly TerrainSpriteLayer terrain;
+		const string TerrainPalette = "terrain";
```

Comment:
Can we define this in the tileset header instead?  That would keep the default and overrides within the same class.